docs/overview.md

```diff
@@ -11,62 +11,59 @@ weight=-99
 
 # Docker Swarm overview
 
-Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
-into a single, virtual Docker host. Because Docker Swarm serves the standard
-Docker API, any tool that already communicates with a Docker daemon can use
-Swarm to transparently scale to multiple hosts. Supported tools include, but
-are not limited to, the following:
+Docker Swarm is native clustering for Docker. You can use it to create a single virtual Docker host from a pool of Docker hosts.
 
-- Dokku
-- Docker Compose
-- Krane
-- Jenkins
+To scale up and run more containers, add more nodes to your Swarm cluster. In production, a Swarm cluster can have 1,000 nodes running 50,000 containers with no performance degradation.
 
-And of course, the Docker client itself is also supported.
+Docker tools are compatible with Swarm. Because the Docker Swarm API is based on the standard Docker Remote API, tools that communicate with a Docker host work with Docker Swarm. These tools include Dokku, Docker Compose, Krane, Flynn, Deis, DockerUI, Shipyard, Drone, Jenkins, the Docker client, and others.
 
 Like other Docker projects, Docker Swarm follows the "swap, plug, and play"
-principle. As initial development settles, an API will develop to enable
-pluggable backends.  This means you can swap out the scheduling backend
-Docker Swarm uses out-of-the-box with a backend you prefer. Swarm's swappable design provides a smooth out-of-box experience for most use cases, and allows large-scale production deployments to swap for more powerful backends, like Mesos.
+principle by supporting pluggable backends. For example, on a large-scale Swarm cluster, you can use Apache Mesos to do scheduling instead of Swarm's built-in tools.
 
-## Understand swarm creation
+## Understand Swarm creation
 
-The first step to creating a swarm on your network is to pull the Docker Swarm image. Then, using Docker, you configure the swarm manager and all the nodes to run Docker Swarm. This method requires that you:
+There are different ways to create a Swarm cluster. This is a simple high-level
```

Comment:
IDT this belongs here.  We have the "How to get and run Swarm that covers this."